JS/app.js

```diff
@@ -25,19 +26,49 @@ photoArray.push(new Photo('img/kitten12.jpg'));
 photoArray.push(new Photo('img/kitten13.jpg'));
 photoArray.push(new Photo('img/kitten14.jpg'));
 
-//etc
+var leftVotes;
+var rightVotes;
 
-var Tracker = function() {
-this.leftPhoto = this.getRandomInt();
-this.rightPhoto = this.getRandomInt();
+var ctx = document.getElementById("myChart").getContext("2d");
+console.dir(Chart)
+var myNewChart = new Chart(ctx).Doughnut([{value: leftVotes, color: 'red'}, {value: rightVotes, color: 'green'}]);
```

Comment:
Don't be afraid to break these complex statements into multiple lines for clarity and readability.